services/database/class.ts

```diff
@@ -62,4 +62,22 @@ async function deleteClass(id: string): Promise<Class> {
     return deletedClass;
 }
 
-export { getClass, getClasses, createClass, deleteClass };
+/**
+ * updateClass takes in id of the class and updates the class information
+ * @param id - classId of the class to be updated
+ * @returns Promise<Class> - Promise with the updated class
+ */
+async function updateClass(
+    id: string,
```

Comment:
@mahadzaryab1 Should we convert `id` to a number prior?

Reply:
@brandonwong368 Can you leave a comment so Mahad can see it later?

Reply:
@mahadzaryab1 bump